app/client/src/pages/VerifyYourSeedPhrase/VerifyYourSeedPhrase.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import {Button, Container, Grid, Header, TextArea} from "semantic-ui-react";
+
+import {useHistory} from "react-router-dom";
+
+import HeaderMenu from "../../Components/HeaderMenu";
+
+function VerifyYourSeedPhrase({seedPhrase}) {
+
+    const [seedPhraseIsCorrect, setSeedPhraseIsCorrect] = React.useState(false);
+
+    const [chosenPhrase, setChosenPhrase] = React.useState([]);
+
+    const [verifyPhraseButtonText, setVerifyPhraseButtonText] = React.useState("Verify Phrase");
+
+    const history = useHistory();
+
+    React.useEffect(() => {
+        if (seedPhraseIsCorrect) {
+            setVerifyPhraseButtonText("Verify Phrase")
+        } else {
+            setVerifyPhraseButtonText("Phrase isn't correct yet!")
+        }
+    }, [seedPhraseIsCorrect]);
+
+    React.useEffect(() => {
+        setSeedPhraseIsCorrect(chosenPhrase.length === 12);
+    }, [chosenPhrase]);
+
+    const handlePhraseClick = seed => {
```

Comment:
:book: Lingo/readability thing, if we can change `seed` to `word`. _Seed Phrase_ generally encompasses the entire mnemonic. Took me a second to process on a straight read that this was not the whole seed being passed, but rather one piece of it ( the clicked word ).

Reply:
I doubted for a while indeed on the variable's name. Let me fix it.

Reply:
This occurs in a few other places. So anywhere seed is referring to a ~6 character string that is a piece of the mnemonic